README.md

```diff
@@ -1,94 +1,24 @@
 [![Run Pytests](https://github.com/eclipse-volttron/volttron-lib-base-historian/actions/workflows/run-test.yml/badge.svg)](https://github.com/eclipse-volttron/volttron-lib-base-historian/actions/workflows/run-test.yml)
-[![documentation](https://img.shields.io/badge/docs-mkdocs%20material-blue.svg?style=flat)](https://VOLTTRON.github.io/volttron-lib-base-historian/)
 [![pypi version](https://img.shields.io/pypi/v/volttron-base-historian.svg)](https://pypi.org/project/volttron-lib-base-historian/)
 
-VOLTTRON base historian framework
+VOLTTRON base historian framework that provide common functions such as caching, error handling, input validation etc.
```

Comment:
Can you add another line that this is a base library for a historian and cannot be used standalone.